frontend/src/components/design-system/other/StepProgress.vue

```diff
@@ -1,13 +1,50 @@
 <script setup lang="ts">
+import { computed } from 'vue';
+
 export interface Props {
   current: number;
 }
 
-defineProps<Props>();
+const props = withDefaults(defineProps<Props>(), {
+  current: 1,
```

Comment:
Tady bych ten default už teda fakt nedával :)

Reply:
Klidně to `withDefaults` úplně vyhoď, jen jak tam bylo:

```
defineProps<Props>();
```

Reply:
Smazáno.